trace_context/HTTP_HEADER_FORMAT.md

```diff
@@ -145,16 +145,43 @@ base16(<TraceOptions>) = 00  // not-sampled
 
 # TraceState field
 
+The `tracestate` HTTP header field conveys information about request position in multiple distributed tracing graphs.
+
 ## Header name
 
 `tracestate`
 
 ## Header value
 
-`vendorName1=opaqueValue1,vendorName2=opaqueValue2`
+This section uses the Augmented Backus-Naur Form (ABNF) notation of [RFC5234](https://tools.ietf.org/html/rfc5234), including the DIGIT rule from that document. It also includes the OWS rule from [RFC7230](https://www.rfc-editor.org/info/rfc7230).
+
```

Comment:
I'd suggest a layman definition here before the `dictionary` RFC-style definition, maybe something like: 

>The `tracestate` header value is a `dictionary` as defined below. The `dictionary` is a series of `dict-members` separated by commas `,`, and a `dict-member` is a key/value pair separated by an equals sign `=`. Spaces and horizontal tabs surrounding `dict-member`s are ignored. There can be a maximum of 128 `dict-member`s in a `dictionary`.
>
>A simple example of a `dictionary` with two `dict-member`s might look like: `vendorname1=opaqueValue1,vendorname2=opaqueValue2`

I know you have that example later, but it might be good to have it up front to give people some context as they're reading through the RFC-style definitions.